### PR TITLE
Syapsification

### PR DESCRIPTION
**Pull request by jeremycarroll**
This PR takes the basic cookiecutter-pypackage, and removes bits that Syapse does not use, and adds stuff that we do.

### CODE REVIEW

**Comment by mianelson** [comment]
pretty cool -- have we used this to create any new projects yet?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* A default [CodeClimate](https://codeclimate.com/) configuration.
+* Linter configurations for PEP8 compliance.
+
+### Changed
+
+* Tox support
+
+### Removed
+
+* Lots of stuff
```
Comment (ganewman42) [inline CHANGELOG.md]: A brief summary of highlights might be helpful.

**Review by ganewman42 — COMMENTED** [review]
Looks good overall, let me read more about the base package and get back to you with additional comments

**Comment by jeremycarroll** [comment]
@mianelson syapse-ontology

**Review by mianelson — APPROVED** [review]

**Pull request merged**